### PR TITLE
284 how to autoupdate landscape

### PR DESCRIPTION
**Pull request by ZeusTheTrueGod**

### CODE REVIEW

**Comment by ZeusTheTrueGod** [comment]
I have not added google analytic yet

**Comment by ZeusTheTrueGod** [comment]
LGTM
Google search console added
Landscapes are added to the autoupdater

File: README.md [hunk 1]
```diff
@@ -97,6 +97,35 @@ alias all='for path in /Users/your-username/dev/{landscapeapp,landscape,lfdl-lan
...
+
+
+## Adding to a google search console
+  Go to the google search console, add a new property, enter the url of the
+  given project, for example, https://landscape.cncf.io
+
+  Next, google will want to verify that it is your site, so you need to choose
+  an `html tag verification` option and copy a secret code from it and put it to
+  the `settings.yml` of a given landscape and commit to the master branch with
+  that change and deploy. The key is named `google_site_veryfication` and it is
```
Comment (dankohn) [inline README.md]: spelling.

Comment (ZeusTheTrueGod) [reply README.md]: @dankohn Is it better now?

Comment (dankohn) [reply README.md]: verification. Note that I’m looking for most of this to change in #298.

**Pull request merged**